src/Microsoft.DotNet.Interactive.ExtensionLab/MermaidKernelExtension.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Html;
+
+namespace Microsoft.DotNet.Interactive.ExtensionLab
+{
+    public class MermaidKernelExtension : IKernelExtension, IStaticContentSource
+    {
+        public string Name => "Mermaid";
+        public Task OnLoadAsync(Kernel kernel)
+        {
+            if (kernel is CompositeKernel compositeKernel)
+            {
+                compositeKernel.Add(new MermaidKernel());
+            }
+
+            kernel.UseMermaid();
+            KernelInvocationContext.Current?.Display(
```

Comment:
It should work, and be cleaner, to send a `DisplayValue` command directly to the kernel, no?

Reply:
makes sense